pkgs/top-level/perl-packages.nix

```diff
@@ -12420,6 +12468,22 @@ let
     };
   };
 
+  MojoIRC = buildPerlPackage {
+    pname = "Mojo-IRC";
```

Comment:
Mojo::IRC is no longer required by Convos.

Reply:
Thx for spotting this. I've removed Mojo::IRC from this PR.